src/unleash_client_wrapper.ts

```diff
@@ -24,6 +24,7 @@ export async function initializeUnleashClient(extensionName: string, serverArgs:
     refreshInterval: serverArgs.ftServerInterval,
     url: serverArgs.ftServerEndPoint,
     strategies: customStrategies,
+    //customHeaders: {"authorization" : ""},
```

Comment:
why is this comment here?

Reply:
it is needed for development, for local testing...